src/main/webapp/script.js

```diff
@@ -562,12 +598,13 @@ function notify(type, id) {
  * @param {boolean} isApproved 
  */
 function mentorApprove(isApproved) {
-  const mentor_id = (new URL(document.location)).searchParams.get('id');
-  console.log(mentor_id);
+  const mentorId = (new URL(document.location)).searchParams.get('id');
+  console.log(mentorId);
   console.log(isApproved);
```

Comment:
remember to remove logs when pushing to master

Reply:
Done!